### PR TITLE
Add test for a feasibility problem

### PR DESCRIPTION
**Pull request by tkoolen**

### CODE REVIEW

File: src/Test/contlinear.jl [hunk 1]
```diff
@@ -1368,6 +1368,57 @@ function linear12test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+# feasibility problem
+function linear13test(model::MOI.ModelLike, config::TestConfig)
+    atol = config.atol
+    rtol = config.rtol
+    # find x
```
Comment (blegat) [inline src/Test/contlinear.jl]: "find x" -> "find x, y"

File: src/Test/contlinear.jl [hunk 2]
```diff
@@ -1368,6 +1368,57 @@ function linear12test(model::MOI.ModelLike, config::TestConfig)
...
+        @test MOI.canget(model, MOI.PrimalStatus())
+        @test MOI.get(model, MOI.PrimalStatus()) == MOI.FeasiblePoint
+
+        @test MOI.canget(model, MOI.VariablePrimal(), MOI.VariableIndex)
+        xsol = MOI.get(model, MOI.VariablePrimal(), x)
+        ysol = MOI.get(model, MOI.VariablePrimal(), y)
+
+        c1sol = 2 * xsol + 3 * ysol
+        @test c1sol >= 1 || isapprox(c1sol, 1.0, atol=atol, rtol=rtol)
+        @test xsol - ysol ≈ 0 atol=atol rtol=rtol
```
Comment (blegat) [inline src/Test/contlinear.jl]: We should check `ConstraintPrimal` too

**Comment by tkoolen** [comment]
Thanks, I've addressed the comments.

**Comment by blegat** [comment]
Thanks !

**Pull request merged**